<launch>
  <param name="calibration_path" value="$(find rpl)/config/camera.xml" />
  <param name="file_path" value="$(find rpl)/config/source_position/"/>
  <param name="slop_ms" type="double"    value="1000.0"/>
  <param name="matrix_path" value="$(find rpl)/config/translate_matrix/"/>

  <node name="camera" pkg="rpl" type="camera.py" output="screen"/>
  <node name="ar" pkg="rpl" type="ar_detector.py" output="screen"/>
  <node name="estimate_position" pkg="rpl" type="estimate_position2.py" output="screen"/>
  <node name="pose_integrator" pkg="rpl" type="pose_integrator.py" output="screen"/>
  <node name="pose_confusion" pkg="rpl"  type="pose_confusion.py" output="screen"/>
  <node name="pose_init" pkg="rpl"  type="pose_init_minimini2.py" output="screen"/>
</launch>
